blak05-cv03

krystofblak05 opened the pull request:
https://eso.vse.cz/~blak05/cv03/index.php

krystofblak05 committed:
blak05-cv03

https://eso.vse.cz/~blak05/cv03/index.php

nvbach91 commented on www/blak05/cv03/index.php:
```diff
@@ -0,0 +1,147 @@
+            $invalidInputs['numCards'] = "We need this information <br />";
+        }else{
+            $numCards = $_POST['numCards'];
+            if(!preg_match('/^[0-9]\z/', $numCards)){
```
nemuzu mit vic jak 9 karet? a muzu prijit bez karet?

krystofblak05 replied:
nahrál jsem v neděli opravenou verzi, je už vše v pořádku? nerad bych přišel o bodík

krystofblak05 committed:
Blak05-cv03

Provedl jsem změnu logiky, pokud uživatel nezadá jméno balíčku, automaticky se doplní 0 a "nemá svůj balíček".  Pokud udělá chybu, a už název balíčku nezmění, a zadá počet karet, automaticky se i tak změní na 0.
